### PR TITLE
`cg` files legend

### PR DESCRIPTION
**Pull request by joaomcteixeira**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ Each sub-directory (one per complex) contains the following files:
 * `XXX_l_u.pdb` : Unbound ligand PDB
 * `XXX_r_b-matched.pdb` : Matched bound receptor PDB
 * `XXX_l_b-matched.pdb` : Matched bound ligang PDB
+* `XXX_r_u_cg.pdb` : Coarse-grain models for the receptor proteins
```
Comment (amjjbonvin) [inline README.md]: Shouldn't there be also a CG model for the ligand?

And may-be change "Coarse-grain" to "Martini coarse-grain"

Comment (joaomcteixeira) [reply README.md]: done

Comment (joaomcteixeira) [reply README.md]: yes, there are also for ligands

Comment (rvhonorato) [reply README.md]: actually martini v2, the new v3 has a different mapping

Comment (joaomcteixeira) [reply README.md]: added

**Review by amjjbonvin — APPROVED** [review]

**Pull request merged**